test/plugins/test_plugins_rules.py

```diff
@@ -1083,6 +1083,13 @@ def handler(wrapped, trigger):
     assert 'threaded' in kwargs
     assert 'output_prefix' in kwargs
     assert 'unblockable' in kwargs
+    assert 'rate_limit' in kwargs
```

Comment:
Along with what I said before about renaming the `rate` attr on callables, this kwarg could be changed to `user_rate_limit` too.

Here I am again, dropping a line note in some mostly-unrelated place because GitHub won't let me go put it [where it belongs](https://github.com/Exirel/sopel/blob/66bae33d60c35dd77460347529bf578f00961812/sopel/plugins/rules.py#L902) on code in `sopel/plugins/rules.py` that's too far from the diff.